packages/web/src/scss/components/Modal/_theme.scss

```diff
@@ -18,19 +18,27 @@ $common-padding-x-tablet: tokens.$space-800;
 // ModalDialog
 $dialog-width: 640px;
 $dialog-width-desktop: 680px;
-$dialog-height: var(--modal-preferred-height-mobile, min-content);
-$dialog-height-tablet: var(--modal-preferred-height-tablet, min-content);
-$dialog-max-height: min(var(--modal-max-height-mobile, 600px), calc(100dvh - #{2 * $padding-y}));
-$dialog-max-height-tablet: min(var(--modal-max-height-tablet, 600px), calc(100dvh - #{2 * $padding-y}));
+$dialog-height: auto;
+$dialog-height-tablet: auto;
+$dialog-max-height: none;
+$dialog-max-height-tablet: none;
 $dialog-text-color: tokens.$text-primary-default;
 $dialog-border-radius: tokens.$radius-200;
 $dialog-background-color: tokens.$background-basic;
 $dialog-shadow: tokens.$shadow-300;
 
+$dialog-scrollable-height: var(--modal-preferred-height-mobile, min-content);
+$dialog-scrollable-height-tablet: var(--modal-preferred-height-tablet, min-content);
+$dialog-scrollable-max-height: min(var(--modal-max-height-mobile, 600px), calc(100dvh - #{2 * $padding-y}));
+$dialog-scrollable-max-height-tablet: min(var(--modal-max-height-tablet, 600px), calc(100dvh - #{2 * $padding-y}));
```

Comment:
how about adding a fallback to the mobile height? And then for desktop
```
var(--modal-max-height-desktop, var(--modal-max-height-tablet, var(--modal-max-height-mobile, 600px)))
```

Reply:
I agree, it just makes more sense to me to do it in the upcoming issue, this one focuses on scrolling.

Reply:
Fixed.